programming/python/libraries.md

```diff
@@ -22,30 +22,31 @@ The following python libraries are installed and available for use in your robot
 * [gmqtt 0.6.10](https://pypi.org/project/gmqtt)
 * [gpg 1.16.0](https://pypi.org/project/gpg)
 * [iniparse 0.5](https://pypi.org/project/iniparse)
-* [j5 0.13.0](https://pypi.org/project/j5)
+* [j5 0.13.1](https://pypi.org/project/j5)
 * [j5-zoloto 0.1.0](https://pypi.org/project/j5-zoloto)
 * [kiwisolver 1.3.2](https://pypi.org/project/kiwisolver)
 * [matplotlib 3.4.1](https://pypi.org/project/matplotlib)
 * [numpy 1.21.2](https://pypi.org/project/numpy)
 * opencv 4.5.2
 * [pandas 1.3.3](https://pypi.org/project/pandas)
-* pkg-resources 57.4.0
+* python3-pkg-resources 57.4.0
 * [pydantic 1.8.2](https://pypi.org/project/pydantic)
 * [pygobject 3.40.1](https://pypi.org/project/PyGObject/)
 * [pyparsing 2.4.7](https://pypi.org/project/pyparsing)
 * [pyquaternion 0.9.9](https://pypi.org/project/pyquaternion)
 * [pyserial 3.5](https://pypi.org/project/pyserial)
 * [pytz 2021.1](https://pypi.org/project/pytz)
+* [pyudev 0.22.0](https://pypi.org/project/pyudev)
 * [pyusb 1.2.1](https://pypi.org/project/pyusb)
 * [pyyaml 5.4.1](https://pypi.org/project/PyYAML/)
 * [rpm 4.16.1.3](https://pypi.org/project/rpm)
 * [setuptools 57.4.0](https://pypi.org/project/setuptools)
 * [six 1.16.0](https://pypi.org/project/six)
-* [sr-robot3 2022.0.1](https://pypi.org/project/sr.robot3/)
+* [sr-robot3 2022.0.2](https://pypi.org/project/sr.robot3/)
 * [toml 0.10.2](https://pypi.org/project/toml)
 * [typing-extensions 3.10.0.0](https://pypi.org/project/typing-extensions)
 * [udiskie 2.3.3](https://pypi.org/project/udiskie)
-* udiskie-locale-en-us 2.3.3
+* python3-udiskie-locale-en-us 2.3.3
```

Comment:
Question: Is the `python3-` prefix here required?

Reply:
Nope

Reply:
Does this apply to `pkg-resources` too?